packages/@uppy/companion/src/server/provider/index.js

```diff
@@ -5,8 +5,7 @@
 const config = require('@purest/providers')
 const dropbox = require('./dropbox')
 const drive = require('./drive')
-const instagram = require('./instagram')
```

Comment:
can we delete any code in the provider implementation?

Reply:
yes, I intend to, was just trying to do it in bits to reduce risk (even though I can't immediately think of any). Reason is, Instagram's Graph API can't easily be tested locally yet (it mandates https), so I'm just removing the key part of the code, to see all goes well. Then I'd proceed to remove the actual code.